From debbdd25835318dc4848488da4dc0acfd27b3332 Mon Sep 17 00:00:00 2001
From: Fabian Vogt <fabian@ritter-vogt.de>
Date: Thu, 6 Jun 2019 20:06:08 +0200
Subject: [PATCH 3/3] Revert "Fix prl replacements if libdir is in
 QMAKE_DEFAULT_LIBDIRS"

This reverts commit d5071a4016ec663f8ec7c89ec7ebabea54b3260f.
---
 mkspecs/features/qt_common.prf | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/mkspecs/features/qt_common.prf b/mkspecs/features/qt_common.prf
index e51aa4766b..843916b3b1 100644
--- a/mkspecs/features/qt_common.prf
+++ b/mkspecs/features/qt_common.prf
@@ -39,10 +39,6 @@ contains(TEMPLATE, .*lib) {
         qt_libdir = $$[QT_INSTALL_LIBS]
     }
     contains(QMAKE_DEFAULT_LIBDIRS, $$qt_libdir) {
-        lib_replace0.match = $$rplbase/lib/
-        lib_replace0.replace = $$qqt_libdir/
-        lib_replace0.CONFIG = path
-        QMAKE_PRL_INSTALL_REPLACE += lib_replace0
         lib_replace.match = "[^ ']*$$rplbase/lib"
         lib_replace.replace =
     } else {
-- 
2.21.0

